src/context.rs

```diff
@@ -235,8 +235,31 @@ impl<'a> Context<'a> {
         disabled == Some(true)
     }
 
+    /// Returns true when a negated environment variable is defined in `env_vars` and is present
+    fn has_negated_env_var(&self, env_vars: &'a [&'a str]) -> bool {
+        env_vars
+            .iter()
+            .any(|env_var| env_var.starts_with('!') && self.get_env(&env_var[1..]).is_some())
```

Comment:
`strip_prefix` can be used here to combine the `starts_with` and `[1..]` part:
```suggestion
        env_vars
            .iter()
            .filter_map(|env_var| env_var.strip_prefix('!'))
            .any(|env_var| self.get_env(env_var).is_some())
```

Reply:
Thank you! Very good recommendation. A lot better to read and probably faster as well.